src/engines/jade/gui/main/main.cpp

```diff
@@ -57,12 +57,12 @@ MainMenu::MainMenu(Module &module, ::Engines::Console *console) : ::Engines::Kot
 	 */
 	getListBox("ListBoxButtons")->setFill("");
 
-	addWidget(getListBox("ListBoxButtons")->createItem("NEW_GAME"));
-	addWidget(getListBox("ListBoxButtons")->createItem("LOAD_GAME"));
-	addWidget(getListBox("ListBoxButtons")->createItem("MINIGAMES"));
-	addWidget(getListBox("ListBoxButtons")->createItem("OPTIONS"));
-	addWidget(getListBox("ListBoxButtons")->createItem("CREDITS"));
-	addWidget(getListBox("ListBoxButtons")->createItem("EXIT"));
+	addWidget(getListBox("ListBoxButtons")->createItem("NEW_GAME"), true);
+	addWidget(getListBox("ListBoxButtons")->createItem("LOAD_GAME"), true);
+	addWidget(getListBox("ListBoxButtons")->createItem("MINIGAMES"), true);
+	addWidget(getListBox("ListBoxButtons")->createItem("OPTIONS"), true);
+	addWidget(getListBox("ListBoxButtons")->createItem("CREDITS"), true);
+	addWidget(getListBox("ListBoxButtons")->createItem("EXIT"), true);
```

Comment:
Why? At that point, the GUI itself isn't shown yet.

Reply:
No idea, that how it worked before :)

Reply:
Huh, am I missing something? You added the parameter `show`, and added `if (show) widget->show()`. So the old behaviour would be the same as `addWidget(..., false);`, no?

Reply:
In *listbox.cpp*, function *createItem*, there was a non optional call to *show* on newly created item. It didn't work right for me, hence I moved that into *addWidget* function.

Reply:
Ah, okay :)